docs/tutorials/frontier/index.md

```diff
@@ -0,0 +1,132 @@
+> crates; it takes Substrate code from github. Take note of this in your `Cargo` files. You _must_
+> use the _matching_ version of dependencies for all of Substrate and Frontier in your project.
+>
+> > [Old tutorial for reference](https://github.com/substrate-developer-hub/frontier-workshop/)
+
+### Frontier Template
+
+To get a stand-alone Frontier template to start, you are encouraged to generate your own. There also
+is a github repo `template` that is available for major Frontier releases pre-generated for you.
+
+#### Generation Script
+
+You can generate a version of the Frontier template by running the
+[generation script](https://github.com/paritytech/frontier/blob/master/.maintain/node-template-release.sh)
+included in Frontier. This can be done starting at **any commit within frontier** to get a specific version/tag.
+
+```bash
+# from the top working dir of Frontier:
+cd .maintain/
+# set the *full file name with .tar.gz extension* for your output file
+./node-template-release.sh TEMPLATE.tar.gz
+# Note the file will be placed in the top level working dir of frontier
+# Move the archive to wherever you like...
+tar xvzf TEMPLATE.tar.gz
+# this unpacks into `frontier-node-template` with all your files
+cd frontier-node-template
+# build the template
+cargo build --release
+```
+
+### Github Template
+
+The Substrate Developer Hub has generated the template using the
+[included release guide](https://github.com/paritytech/frontier/blob/master/docs/node-template-release.md)
+, and intends to update with major releases of Frontier moving forward.
+
+You can `use` the pre-generated template or `fork` it from here:
+https://github.com/substrate-developer-hub/frontier-node-template/ .
+
+### Testing Ethereum Functionality
+
+To test the template, please see the [included
+README](https://github.com/paritytech/frontier/blob/master/template/README.md) that details the
+proper Ethereum account use, as well as deploying and interacting with
+EVM contracts.
+
+## Frontier Overview & Architecture Diagrams
+
+Here are a few helpful diagrams to help illustrate how the Frontier EVM and Ethereum RPC plug into
+your Substrate FRAME runtime.
+
+### EVM Pallet Runtime Configuration
+
+The Ethereum Virtual Machine (EVM) is a sandboxed virtual stack machine that is implemented in the
+EVM pallet. The EVM is responsible for executing Ethereum contract bytecode of smart contracts,
+typically written in a high level language like Solidity, then compiled to EVM bytecode.
+
+![architecture diagram](assets/tutorials/frontier/pallet-evm.png)
+
+### Ethereum Pallet
+
+The Ethereum pallet is responsible for storing Ethereum-formatted blocks, transaction receipts, and transaction statuses.
+
+The biggest challenge here is that the Frontier project does not use the same, published, Substrate
+crates; it takes Substrate code from github. For this tutorial, I've made a special branch of
```

Comment:
> Link to branch?

Reply:
https://github.com/substrate-developer-hub/substrate-developer-hub.github.io/pull/1087 to remove for now. Will detail more in followup #1085